src/app/shared/image/image-upload.component.css

```diff
@@ -24,7 +24,6 @@
 }
 .new-image label {
   padding: 6px 10px;
-  margin-bottom: 0;
```

Comment:
.button now has `margin: 0` so doesn't have to clear label's `margin-bottom: 3px`